fili-core/src/main/java/com/yahoo/bard/webservice/table/BasePhysicalTable.java

```diff
@@ -23,7 +24,7 @@
 public abstract class BasePhysicalTable implements PhysicalTable {
     private static final Logger LOG = LoggerFactory.getLogger(BasePhysicalTable.class);
 
-    private final String name;
+    private final TableName name;
```

Comment:
It should not be necessary, and if we do, will variable type `String` have name `string`?